src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.2/traits/trait_block_properties.json",
+  "title": "Block properties",
+  "description": "Block properties for style formatting content elements",
+  "type": "object",
+  "properties": {
+    "dropcap": {
+      "title": "Dropcap",
+      "description": "Style the first letter or first word with a dropcap",
+      "type": "string",
+      "enum": ["letter", "word"]
```

Comment:
I think per @awong7 we're not going to support `word` type

Reply:
@yury-sannikov in that case, should dropcap be a bool?